docs/src/userguide/fusedtypes.rst

```diff
@@ -162,24 +302,61 @@ by calling.
 Indexing
 --------
 
-You can index functions with types to get certain specializations, i.e.::
+You can index functions with types to get certain specializations, i.e.:
 
-    cfunc[cython.p_double](p1, p2)
+.. tabs::
 
-    # From Cython space
-    func[float, double](myfloat, mydouble)
+    .. group-tab:: Pure Python
 
-    # From Python space
-    func[cython.float, cython.double](myfloat, mydouble)
+        .. literalinclude:: ../../examples/userguide/fusedtypes/indexing.py
+            :caption: indexing.py
 
-If a fused type is used as a base type, this will mean that the base type is the
-fused type, so the base type is what needs to be specialized::
+    .. group-tab:: Cython
 
-    cdef myfunc(A *x):
-        ...
+        .. literalinclude:: ../../examples/userguide/fusedtypes/indexing.pyx
+            :caption: indexing.pyx
+
+Indexed functions can be called directly from Python:
+
+.. code-block:: pycon
+
+    >>> import cython
+    >>> import indexing
+    cfunc called: double 5.0 double 1.0
+    cpfunc called: float 1.0 double 2.0
+    func called: float 1.0 double 2.0
+    >>> indexing.cpfunc[cython.float, cython.float](1, 2)
+    cpfunc called: float 1.0 float 2.0
+    >>> indexing.func[cython.float, cython.float](1, 2)
+    func called: float 1.0 float 2.0
+
+If a fused type is used as a component of a more complex type
+(for example a pointer to a fused type, or a memoryview of a fused type),
+then you should index the function with the individual component and
+not the more full argument type:
```

Comment:
```suggestion
not the full argument type:
```

You commented on this and then deleted the comment (I think). But you were right - what I'd proposed included an extra word that didn't make sense :). Sorry!

Reply:
Hm I removed the comment because I checked it against spell checker and checker does not complain about it so I considered is as correct in the end...